### DISCUSSION TITLE
Repo Settings Tab

### DISCUSSION THREAD

**Discussion by CuzImBisonratte** [community/community#10681] [Mobile] [answered]
I would like it very much, if in repos there would be an settings tab to like change visibility, set up pages and so on, on the go.

**Comment by taki-on** [comment, marked as answer]
Thank you for your feedback @CuzImBisonratte ! Supporting repo settings on mobile is not on our roadmap, but I will forward this to the team so that we can discuss further within the team. Thank you!
